schemas/styles/csl-choose.rnc

```diff
@@ -64,6 +64,38 @@ div {
       attribute variable {
         list { variables+ }
       }
+      |
+      condition.atts.complex.dates
+      |
+      condition.atts.complex.general
+
+  
+  # Tests that require two attributes 
+  # Dates
+  condition.atts.complex.dates =
+    ## specifies the variable to be tested
+    attribute tested { 
+      list { variables.dates+ }
+     },
+     (date-precision | date-range)+
```

Comment:
> Currently, the date test requires date-precision and date-range. How can one of them be made optional?

```suggestion
     (date-precision, date-range?)
```

... requires precision, leaves range optional.

Reply:
Oh I forgot to check the box. I think `(date-precision | date-range)+` already means that at least one of them is needed. The other is optional. I don't think we should prescribe which one is required